.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Please put this in a separate PR after this one; we should get OpenSSL update out of the way first.